docker-compose.yml

```diff
@@ -1,9 +1,7 @@
 version: "3.3"
 services:
   iroha:
-    build:
-      context: .
-    image: iroha:debug
+    image: iroha1/iroha:iroha2-dev
```

Comment:
The use of local images here was intentional I don't think we should use the ones from Docker hub for 2 reasons:
1. In CI it is needed to check current docker setup and it is not published yet (and there is nor reason to publish it before checked)
2. The docker compose setup is mainly meant for debugging our issues locally so again - we want to use local containers.

Users of Iroha2 will be able to use images they need with their custom tags and for this they are free to take our docker compose as an example.

Reply:
Ah, I see you build it locally and don't push

Reply:
It literally uses current images. Building locally takes lots of time, so that is why it would be better for users to take latest image from registry

Reply:
So I assume you do it to cash the image. But why do so? For example in new commits we can change something in the code and image will not be valid.

Reply:
I guess they can update it by `docker pull`. In some time we will have normal versioning and image tagging, but for now i would bet that its better.

Reply:
Ok, we discussed it on a call, seem I misunderstood. Everything is ok.